core/themes/basis/css/component/cards.css

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ * Card listing view on the default home page.
+ */
+
+.l-top .block-views-promoted-cards-block {
+  margin-top: -2rem;/* Negative top margin collapses spacing under header */
```

Comment:
This fixes a bug in basis (that we may fix shortly) should we add a @todo comment?